### ISSUE TITLE
Spike: Usage of JSS in Porsche Design System

### ISSUE THREAD

**Issue by denyo** [porsche-design-system/porsche-design-system#959]
**Scope**
Test JSS instead of classic (S)CSS within one component to validate the improvements on productivity of component creation, component size/loading performance, maintenance.

**Notes**
General advantages
- Rendered styles are cached. Compilation and DOM Rules creation happens only once
- Only styles which are currently in use on your screen are also in the DOM
- Dynamic styles, e.g. theme support to only render light theme or dark theme styles into DOM
- Unreadable generic classnames to prevent copying/stealing CSS and markup
- Type checking of classnames (no typos, less unused styles)
- Reusable styles by merging JS objects across components
- Reduced redundancy without BEM syntax
- No need for `prefix` function

**Out of Scope**  
List topics that are not part of this task.

**Notes**  
Share any further information that is relevant for processing.
- https://cssinjs.org/?v=v10.4.0
- https://github.com/Justin-Palmer/stencil-css-in-js

Subtasks
---
- [ ] Task
